### DISCUSSION TITLE
Possible to add anonymous marks?

### DISCUSSION THREAD

**Discussion by smilingpeanut** [maize-tech/laravel-markable#9] [Ideas]
Tying marks to users is very handy, but it would also be useful to allow marks to be recorded without them being tied to a particular user. Is that possible?

**Comment by riccardodallavia** [comment]
Hi @smilingpeanut!

Could you please provide us with a use case where your suggestion could be useful?
At the moment I feel like it would be out of scope for the package, but please let us know if you do not agree.

Thank you!

**Reply by smilingpeanut** [reply]
Certainly, and thank you for the reply!

We have a radio and podcast listening app. Users do not have to log in to be able to listen to them. We want to display episodes that are popular by querying episode models with the most likes, but don't want to force users to log in just to register a like for an episode.

Hope that makes sense, and thank you!

**Reply by estudiobarragan** [reply]
Perhaps an inelegant solution is: Create a user called guest (without password or email) which is the user that you associate with the person who makes the like

**Reply by riccardodallavia** [reply]
@smilingpeanut interesting, that would be indeed a nice improvement for the package.
Are you willing to submit us a pull request for this?
